gobblin-api/src/main/java/gobblin/converter/Converter.java

```diff
@@ -121,4 +123,8 @@ public State getFinalState() {
     outputStream = outputStream.doOnComplete(this::close);
     return inputStream.withRecordStream(outputStream, outputSchema);
   }
+
+  public ConverterInitializer getInitializer(State state, WorkUnitStream workUnits, int branches, int branchId) {
+    return null;
```

Comment:
Maybe return a Noop `ConverterInitializer` instead? That avoids null checks and NPEs.